### DISCUSSION TITLE
alerts

### DISCUSSION THREAD

**Discussion by pablomontero03** [datalust/seq-tickets#2027] [Help]
In my SEQ the alerts option does not appear, it is the Seq version 2020.5.5163, a bell icon should appear but it does not appear

![image](https://github.com/datalust/seq-tickets/assets/79713314/95c9dae9-ef10-4e80-ad05-6fd9d55a005d)

**Comment by liammclennan** [comment]
The alerts feature you are referring to did not appear until [2021.3](https://blog.datalust.co/seq-2021-3-has-shipped/).

Please note that 2020.5 is a long way out of support. https://docs.datalust.co/docs/version-support-policy
